src/redux/thunks/previewCluster.ts

```diff
@@ -181,7 +181,11 @@ async function loadCustomResourceObjects(kc: KubeConfig, customResourceDefinitio
       // retrieve objects using latest version name
       // @ts-ignore
       k8sApi
-        .listClusterCustomObject(r.content.spec.group, findDefaultVersion(r) || 'v1', r.content.spec.names.plural)
+        .listClusterCustomObject(
+          r.content.spec.group,
```

Comment:
should we use conditional chaining on `r.content` as well? or check if this is not undefined